### DISCUSSION TITLE
Do you have a similar Azure library/reference or plans to make one? Or could we use the design to create a similar architecture for Azure? Any inherited differences that make this not applicable in Azure?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#283] [Help] [answered]
Do you have a similar Azure library/reference or plans to make one? Or could we use the design to create a similar architecture for Azure? Are there inherited differences in Azure that make the design not applicable in Azure?

**Comment by pete0emerson** [comment, marked as answer]
We currently don't have a similar library for Azure or GCP. Although we'd like to eventually support more than AWS, it's not currently on our roadmap. Following the same design pattern for other platforms that Terraform supports is certainly possible, but as you can imagine, it's a lot of work. We want to make sure we're successful with AWS before spreading out to other platforms.
